controller/work_item_link_blackbox_test.go

```diff
@@ -103,8 +103,10 @@ func (s *workItemLinkSuite) cleanup() {
 	require.Nil(s.T(), db.Error)
 	db = db.Unscoped().Delete(&link.WorkItemLinkCategory{Name: "test-user"})
 	require.Nil(s.T(), db.Error)
-	db = db.Unscoped().Delete(&space.Space{ID: s.userSpaceID})
-	require.Nil(s.T(), db.Error)
+	if s.userSpaceID != uuid.Nil {
```

Comment:
@baijum yes this may not look good, but we could find this way to keep `system.space` intact for other tests to make use of.